mmv1/products/networkservices/Gateway.yaml

```diff
@@ -206,3 +207,12 @@ properties:
       This feature only applies to gateways of type 'SECURE_WEB_GATEWAY'.
     item_type:
       type: String
+  - name: 'routingMode'
+    type: Enum
+    immutable: true
+    description: |
+      The routing mode of the Gateway. This field is configurable only for gateways of type SECURE_WEB_GATEWAY. This field is required for gateways of type SECURE_WEB_GATEWAY.
+    custom_flatten: 'templates/terraform/custom_flatten/network_services_gateway_routing_mode.go.tmpl'
```

Comment:
Is the `custom_flatten` needed here? It causes the terraform plan not empty.

Reply:
It's not